x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/physical/EsQueryExec.java

```diff
@@ -42,6 +43,7 @@ public class EsQueryExec extends LeafExec implements EstimatesRowSize {
     );
 
     public static final EsField DOC_ID_FIELD = new EsField("_doc", DataType.DOC_DATA_TYPE, Map.of(), false);
+    public static final EsField SCORE_FIELD = new EsField(MetadataAttribute.SCORE, DataType.DOUBLE, Map.of(), false);
```

Comment:
I found it is a bit confusing to see `SCORE` as a `DOUBLE` here, however it looks like it is a `FLOAT` in `PositionToXContent` and `ResponseValueUtils`.

Reply:
I made it a DOUBLE because this way it all plays nicely when working with EVAL. For example, using a float would make the following query fail:
> FROM collection METADATA _score | WHERE text_field MATCH \"hello friend\" | EVAL lscore = log(_score) | KEEP text_field, lscore

this is because, AFAIU, float, while supported in the compute engine, is not supported in other parts of ES|QL.